packages/manager/apps/telecom/src/app/telecom/pack/migration/meeting/pack-migration-meeting.controller.js

```diff
@@ -0,0 +1,135 @@
+import get from 'lodash/get';
+
+export default class TelecomPackMigrationMeetingCtrl {
+  /* @ngInject */
+  constructor(
+    $scope,
+    $translate,
+    OvhApiConnectivityEligibility,
+    OvhApiConnectivityEligibilitySearch,
+    TucPackMigrationProcess,
+    TucToast,
+  ) {
+    this.$scope = $scope;
+    this.$translate = $translate;
+    this.OvhApiConnectivityEligibility = OvhApiConnectivityEligibility;
+    this.OvhApiConnectivityEligibilitySearch = OvhApiConnectivityEligibilitySearch;
+    this.TucPackMigrationProcess = TucPackMigrationProcess;
+    this.TucToast = TucToast;
```

Comment:
use instance notation not Classes one:

```suggestion
    this.ovhApiConnectivityEligibility = OvhApiConnectivityEligibility;
    this.ovhApiConnectivityEligibilitySearch = OvhApiConnectivityEligibilitySearch;
    this.tucPackMigrationProcess = TucPackMigrationProcess;
    this.tucToast = TucToast;
```